packages/superset-ui-legacy-plugin-chart-table/package.json

```diff
@@ -28,12 +28,14 @@
     "access": "public"
   },
   "dependencies": {
-    "d3": "^3.5.17",
-    "datatables.net-bs": "^1.10.15",
-    "dompurify": "^2.0.6",
-    "prop-types": "^15.6.2"
+    "@superset-ui/query": "^0.12.8",
```

Comment:
Move this to `peerDependencies`. 
There are several `@superset-ui` packages that use singleton so we have to ensure it has only one copy, similar to how `react` works. (Although some packages do not use singleton, I made all of them always `peerDependencies` to avoid confusion whether this should be `dependencies` or `peerDependencies`)

Reply:
Got it